docker-compose.yml

```diff
@@ -9,47 +8,15 @@ services:
       dockerfile: ops/docker/Dockerfile
     environment:
       FEATURE_FLAGR_URL: "http://flagr:18000"
-      DB_HOST: "db"
     volumes:
       - .:/app
-      
     depends_on:
-      - db
       - flagr
 
   flagr:
     image: ghcr.io/openflagr/flagr
     environment:
-      FLAGR_DB_DBDRIVER: mysql
```

Comment:
**question:** I assume flagr falls back to file based / SQLite etc within the container?

Reply:
Yep